roadmap/2022.md

```diff
@@ -186,18 +186,18 @@ Meilisearch releases:
 
 Maintenance:
 - [x] Enhance `pagy` documentation in the meilisearch-rails.
-- [ ] Improve performance in meilisearch-rails by reducing the payload size with `attributesToRetrieve` automatically.
+- [x] _(done, but not released, waiting more test cov, https://github.com/meilisearch/meilisearch-rails/pull/228)_ ~Improve performance in meilisearch-rails by reducing the payload size with `attributesToRetrieve` automatically.~
 - [ ] Wrap up the Java SDK and release the new enhanced version with the addition of support to
-  - [ ] v0.28 Meilisearch version
+  - [x] v0.28 Meilisearch version
   - [ ] v0.29 Meilisearch version
   - [x] Add analytics setup to the SDK.
 
 New projects:
 - [ ] Implement packer.io to build Meilisearch images for the cloud providers.
 - [ ] Build the action plan to create the MVP of the new Laravel-Scout-Integration-Extended on our side.
 - [ ] "Docscraper 2.0":
-  - [ ] Do the complete research around autocomplete.js and docsearchbar v2 support with our JavaScript client.
-  - [ ] Provide the action plan to add support for those libraries in case they are incompatible.
+  - [x] Do the complete research around autocomplete.js and docsearchbar v2 support with our JavaScript client.
+  - [x] Provide the action plan to add support for those libraries in case they are incompatible.
```

Comment:
@bidoubiwa, is that correct to say that you were able to finish these two items? Cause I think so!